clang/lib/Tooling/DependencyScanning/DependencyScanningWorker.cpp

```diff
@@ -744,14 +746,17 @@ bool DependencyScanningWorker::scanDependencies(
   sanitizeDiagOpts(*DiagOpts);
   auto Diags = CompilerInstance::createDiagnostics(*FS, *DiagOpts, &DC,
                                                    /*ShouldOwnClient=*/false);
+  bool canonicalizeDefs =
```

Comment:
```suggestion
  bool CanonicalizeDefs =
```

Reply:
But maybe I'd prefer if we just did the check in the place of use. Propagating these gets very tedious, see #128959.